internal/impl/azure/output_adls.go

```diff
@@ -0,0 +1,158 @@
+// Copyright 2024 Redpanda Data, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package azure
+
+import (
+	"context"
+	"fmt"
+
+	dlservice "github.com/Azure/azure-sdk-for-go/sdk/storage/azdatalake/service"
+	"github.com/redpanda-data/benthos/v4/public/service"
+)
+
+const (
+	// Azure Data Lake Storage Output Fields
+	adlsFieldFilesystem = "filesystem"
+	adlsFieldPath       = "path"
```

Comment:
Not sure what `s` in `adls` stands for. The prefix here should be something like `adlo`, where `o` stands for output in case we'll want to add a matching input in the future.

Reply:
It stands for "Azure Data Lake Storage"

Reply:
Ohhh, OK. `dlo` is fine, thanks! It's not super-consistent yet :)